Sources/Citadel/SFTP/SFTPServerInboundHandler.swift

```diff
@@ -0,0 +1,173 @@
+                    )
+                )
+            }.whenFailure { _ in
+                context.channel.close(promise: nil)
+            }
+        case .read(let command):
+            print(command)
+        case .write(let command):
+            var handle = command.handle
+            
+            guard
+                let id: UInt32 = handle.readInteger(),
+                handle.readableBytes == 0,
+                let file = files[id]
+            else {
+                logger.error("bad SFTP file andle")
+                return
+            }
+            
+            let promise = context.eventLoop.makePromise(of: SFTPStatusCode.self)
+            file.write(command.data, atOffset: command.offset, promise: promise)
+            promise.futureResult.flatMap { status in
+                context.channel.writeAndFlush(
+                    SFTPMessage.status(
+                        SFTPMessage.Status(
+                            requestId: command.requestId,
+                            errorCode: status,
+                            message: "",
+                            languageTag: "EN"
+                        )
+                    )
+                )
+            }.whenFailure { _ in
+                context.channel.close(promise: nil)
+            }
+        case .handle(let command):
+            print(command)
+        case .status(let command):
+            print(command)
+        case .data(let command):
+            print(command)
+        case .mkdir(let command):
+            print(command)
+        case .stat(let command):
+            let promise = context.eventLoop.makePromise(of: SFTPFileAttributes.self)
+            promise.completeWithTask {
+                try await self.delegate.fileAttributes(atPath: command.path)
+            }
+            
+            promise.futureResult.whenSuccess { attributes in
+                context.writeAndFlush(
+                    NIOAny(SFTPMessage.attributes(
+                        .init(
+                            requestId: command.requestId,
+                            attributes: attributes
+                        )
+                    )),
+                    promise: nil
+                )
+            }
+        case .lstat(let command):
+            let promise = context.eventLoop.makePromise(of: SFTPFileAttributes.self)
+            promise.completeWithTask {
+                try await self.delegate.fileAttributes(atPath: command.path)
+            }
+                
+            promise.futureResult.whenSuccess { attributes in
+                context.writeAndFlush(
+                    NIOAny(SFTPMessage.attributes(
+                        .init(
+                            requestId: command.requestId,
+                            attributes: attributes
+                        )
+                    )),
+                    promise: nil
+                )
+            }
+        case .attributes(let command):
+            print(command)
```

Comment:
same here for print statement

Reply:
this is a server-response type, so it cannot be received by a server

Reply:
This should throw an error